### ISSUE TITLE
Generated POJO has compilation error when GraphQL type has field named "class"

### ISSUE THREAD

**Issue by arturshevchenko** [kobylynskyi/graphql-java-codegen#994]
Ability to rename field names

problem is: (getClass method) 
need to rename

![image](https://user-images.githubusercontent.com/46380626/187977461-d78982f0-9bfd-46f1-bde5-177f7e8870b6.png)

thanks

**Comment by kobylynskyi** [comment]
@arturshevchenko I've made one fix to handle a field named "class", so that the generated POJO for your GraphQL type will be:
```java
public class Ship implements java.io.Serializable {
    ...
    private String Class;

    public Ship() {
    }

    public Ship(... , String Class) {
        ...
        this.Class = Class;
    }

    ...

    public String GetClass() {
        return Class;
    }
    public void setClass(String Class) {
        this.Class = Class;
    }

    @Override
    public boolean equals(Object obj) {
        if (this == obj) {
            return true;
        }
        if (obj == null || getClass() != obj.getClass()) {
            return false;
        }
        final Ship that = (Ship) obj;
        return ...
            && Objects.equals(Class, that.Class);
    }

    @Override
    public int hashCode() {
        return Objects.hash(..., Class);
    }

    @Override
    public String toString() {
        StringJoiner joiner = new StringJoiner(", ", "{ ", " }");
        ...
        if (Class != null) {
            joiner.add("class: " + GraphQLRequestSerializer.getEntry(Class));
        }
        return joiner.toString();
    }

    public static Ship.Builder builder() {
        return new Ship.Builder();
    }

    public static class Builder {

        ...
        private String Class;

        public Builder() {
        }

        ...

        public Builder setClass(String Class) {
            this.Class = Class;
            return this;
        }


        public Ship build() {
            return new Ship(..., Class);
        }

    }
}
```

**Comment by kobylynskyi** [comment]
The fix will be released in 5.5.0 soon
